### PR TITLE
SimpleBank Online

### PR DESCRIPTION
**Pull request by doomspork**
Part 2 of SimpleBank.  Adding a basic JSON API.  This is still a WIP.

### CODE REVIEW

**Comment by doomspork** [comment]
I'm going to rebase and clean this up.  Any additional changes you can think of?

File: simple_bank_online/lib/simple_bank_online/router.ex [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  post "/accounts" do
+  end
+
+  get "/accounts/:id" do
+  end
+
+  post "/accounts/:id/deposits" do
+  end
+
+  post "/accounts/:id/withdrawals" do
```
Comment (pjhampton) [inline simple_bank_online/lib/simple_bank_online/router.ex]: It really wouldn't surprise me if this was how my bank had it in their account management system

**Review by pjhampton — APPROVED** [review]

**Comment by pjhampton** [comment]
Looks great so far - are you leaving auth to another day?

File: simple_bank_online/test/simple_bank_online_test.exs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+defmodule SimpleBankOnlineTest do
+  use ExUnit.Case
```
Comment (doomspork) [inline simple_bank_online/test/simple_bank_online_test.exs]: Remove this test.

**Comment by doomspork** [comment]
@pjhampton yes sir!  I thought we could build on this banking example for other lessons.

In fact, I'd like to have a couple different "projects" in Homework that you incrementally build on.  I would personally find that more engaging.

Thoughts @elixirschool/developers?

**Review by pragmaticivan — APPROVED** [review]
ignore my review, just disabling the notification.

**Pull request closed without merging**